lib/dfe/analytics/entity_table_check_job.rb

```diff
@@ -27,16 +27,47 @@ def perform
 
       def entity_table_check_data(model)
         checksum_calculated_at = Time.now.in_time_zone(TIME_ZONE).iso8601(6)
+        adapter_name = ActiveRecord::Base.connection.adapter_name.downcase
+        row_count, checksum = fetch_checksum_data(model, checksum_calculated_at, adapter_name)
+
```

Comment:
Should be say:
``` Ruby
if adapter_name == 'postgresql' && Rails.env.production?
   Rails.logger.info('Only Postgres databases supported on PRODUCTION')
   return
end 
```

Reply:
@asatwal I'm not sure I follow the logic here - would we want the log message in this scenario or only if it is not postgres and production?

Reply:
@ericaporter  - Apologies I got that wrong.. It's NOT Postgres - Corrected above.